src/mjolnir/ingest_transit.cc

```diff
@@ -394,20 +406,30 @@ float get_stop_pair_dist(const gtfs::Stop& stop_connect,
 }
 
 // return dangling stop_pairs, write stop data from feed
-bool write_stop_pair(Transit& tile,
-                     const tile_transit_info_t& tile_info,
-                     const feed_object_t& feed_trip,
-                     const gtfs::Feed& feed,
-                     const std::unordered_map<feed_object_t, GraphId>& platform_node_ids,
-                     unique_transit_t& uniques) {
+bool write_stop_pair(
+    Transit& tile,
+    const tile_transit_info_t& tile_info,
+    const feed_object_t& feed_trip,
+    const gtfs::Feed& feed,
+    const std::unordered_map<feed_object_t, GraphId>& platform_node_ids,
+    unique_transit_t& uniques,
+    const google::protobuf::RepeatedPtrField<valhalla::mjolnir::Transit_Node>& tile_nodes) {
```

Comment:
just adding another argument for this tile's node objects

strange that clang-format decided to align the args differently